internal/actor/permission.go

```diff
@@ -0,0 +1,9 @@
+package actor
+
+//go:generate enumerator -type Permission -linecomment
+type Permission uint8
+
+const (
+	None  Permission = iota //none
+	Admin                   // admin
```

Comment:
PermissionNone and PermissionAdmin to make it more clear? (and then `-trimprefix` on the enumerator call)

Reply:
I won't make the permissions/role comment. This rename can also be picked up when we change these later